resources/base/CancelCommittedOrderConfirmation.view.bxb

```diff
@@ -0,0 +1,68 @@
+confirmation-view {
```

Comment:
This duplicates @kangawoo44 's PR about fixing deprecation warnings

Reply:
coordinated with Woo